src/test/java/seedu/duke/storage/FoodListEncoderTest.java

```diff
@@ -45,8 +45,8 @@ void encodeFoodList_withMultipleFood_expectMultipleFoodRecorded() throws DukeExc
         fl.addFood(strawberry);
         ArrayList<String> encodedFoodLists = FoodListEncoder.encodeFoodList(fl);
         ArrayList<String> expectedOutput = new ArrayList<>();
-        expectedOutput.add("peanuts « 11/11/2023 « 0.0 « unit « OTHERS");
-        expectedOutput.add("strawberries « 31/05/2023 « 0.0 « unit « OTHERS");
+        expectedOutput.add("  «n  peanuts  «e  11/11/2023  «q  0.0  «u  unit  «c  OTHERS");
```

Comment:
using <<n <<e ... is clearer and better than simply <<, which we still have to figure out which is what attribute using its order and parameter name.

Reply:
Good change, improves clarity!